components/SideNavbar/SideNavbarCategoryList.tsx

```diff
@@ -32,7 +32,7 @@ export const SideNavbarCategoryList: FC<{
     <ul
       data-custom="restrict-click-outside"
       ref={listRef}
-      className="relative w-full max-h-full flex flex-col sc overflow-y-scroll purpleScrollbar pr-4"
+      className="relative w-full max-h-full flex flex-col sc overflow-y-scroll scrollColorDarkMobile pr-4"
```

Comment:
The white scrollbar isn't visible in light mode.

<img width="431" alt="Screenshot 2024-01-03 at 10 52 27 AM" src="https://github.com/rupali-codes/LinksHub/assets/93012310/8a7e9364-8d97-4d6f-b9fb-8f7ce6332ec3">


```suggestion
      className="relative w-full max-h-full flex flex-col sc overflow-y-scroll purpleScrollbar dark:scrollColorDarkMobile pr-4"
```

I'm not sure if this suggestion would work, but please make sure to test both light and dark themes.

Reply:
Okay. Let me fix it.

Reply:
I was able to fix the issue.